src/components/Room/Room.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import { set } from 'lodash';
+import ToggleSwitch from '../../components/ToggleSwitch/ToggleSwitch';
+import RoomSlider from './RoomSlider.js';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```

Comment:
since using styled components, Room.css is not used anymore, so better remove it!